examples/kmm-sample/androidApp/src/main/java/io/realm/example/kmmsample/androidApp/MainActivity.kt

```diff
@@ -52,17 +53,11 @@ class MainActivity : AppCompatActivity() {
 
         numATV.addTextChangedListener(textWatcher)
         numBTV.addTextChangedListener(textWatcher)
-    }
 
-    override fun onResume() {
-        super.onResume()
-        registration = Calculator.listen {
-            countTV.text = "History count: ${Calculator.history().size}"
+        lifecycleScope.launch {
```

Comment:
I think there's a value in keeping it inside the `onResume` callback to demonstrate the cancel capability?

Reply:
But `lifecycleScope` automatically handles the cancelation?

Reply:
After all it's also beneficial to demonstrate both approaches, so let's keep the `lifecycleScope` 👍